src/main/java/io/github/syst3ms/skriptparser/tags/TagColor.java

```diff
@@ -0,0 +1,42 @@
+package io.github.syst3ms.skriptparser.tags;
+
+import io.github.syst3ms.skriptparser.Parser;
+import io.github.syst3ms.skriptparser.registration.tags.ContinuousTag;
+import io.github.syst3ms.skriptparser.util.color.ConsoleColors;
+
+import java.util.Locale;
+import java.util.Optional;
+
+public class TagColor implements ContinuousTag {
+
+    static {
+        Parser.getMainRegistration().addTag(TagColor.class);
+    }
+
+    private ConsoleColors color;
+
+    @Override
+    public boolean init(String key, String[] parameters) {
+        final Optional<ConsoleColors> optional;
+        if (key.equalsIgnoreCase("color") && parameters.length != 0)
+            optional = ConsoleColors.search(parameters[0].toUpperCase(Locale.ROOT));
+        else
+            optional = ConsoleColors.search(key);
+
+        if (optional.isEmpty())
+            return false;
+
+        color = optional.get();
+        return color != ConsoleColors.RESET;
+    }
```

Comment:
Please use brackets (``{}``) for your conditions

Reply:
Yessir